spec.html

```diff
@@ -22060,13 +22085,17 @@ <h1>ResolveExport ( _exportName_, _resolveSet_ ) Concrete Method</h1>
                 1. Return ResolvedBinding Record { [[Module]]: _module_, [[BindingName]]: _e_.[[LocalName]] }.
             1. For each ExportEntry Record _e_ in _module_.[[IndirectExportEntries]], do
               1. If SameValue(_exportName_, _e_.[[ExportName]]) is *true*, then
-                1. Assert: _module_ imports a specific binding for this export.
-                1. Let _importedModule_ be ? HostResolveImportedModule(_module_, _e_.[[ModuleRequest]]).
-                1. Return _importedModule_.ResolveExport(_e_.[[ImportName]], _resolveSet_).
+                1. If _e_.[[ImportName]] is `"*"`, then
+                  1. NOTE: _module_ does not provide the direct binding for this export.
+                  1. Return ResolvedBinding Record {[[Module]]: _e_.[[ModuleRequest]], [[BindingName]]: `"*namespace*"`}.
+                1. Else,
+                  1. Assert: _module_ imports a specific binding for this export.
+                  1. Let _importedModule_ be ? HostResolveImportedModule(_module_, _e_.[[ModuleRequest]]).
+                  1. Return _importedModule_.ResolveExport(_e_.[[ImportName]], _resolveSet_)
             1. If SameValue(_exportName_, `"default"`) is *true*, then
               1. Assert: A `default` export was not explicitly defined by this module.
               1. Return *null*.
-              1. NOTE: A `default` export cannot be provided by an `export *`.
+              1. NOTE: A `default` export cannot be provided by an `export * from "mod"` declaration.
```

Comment:
are we sure here that we don't want to mention both forms?

Reply:
seems like we should have both, for sure

Reply:
I agree! will submit fix.